include/entity/triangularMesh.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ENTITY_TRIANGLEMESH_H
+#define ENTITY_TRIANGLEMESH_H
+
+#include <array>
+#include <vector>  
+#include <graphic/color.h>
+#include <geometry/surfaceIntersection.h>
+#include <geometry/ray.h>
+#include <entity/box.h>  
+
+class TriangularMesh : public Box {
+private:
+    int numTriangles;
+    int numVertices;
```

Comment:
Acho válido deixar mais verboso

```suggestion
    int numberOfTriangles;
    int numberOfVertices;
```